### ISSUE TITLE
Cannot import non-existent remote object

### ISSUE THREAD

**Issue by JoschaLaubach** [grafana/terraform-provider-grafana#251]
### Terraform Version
* Terraform: 1.0.3
* Terraform Grafana Provider: 1.13.3
* Grafana: v8.0.6

### Affected Resource(s)
Please list the resources as a list, for example:
* grafana_dashboard

### Terraform Configuration Files
```hcl
terraform {
  required_providers {
    grafana = {
      source = "grafana/grafana"
      version = "1.13.3"
    }
  }
  required_version = ">= 1.0"
}

provider "grafana" {
  url    = "https://grafana.url-to-our-grafana/"
  auth   = var.auth
  org_id = var.org_id
}

resource "grafana_folder" "example" {
    title       = var.title
}

resource "grafana_dashboard" "example" {
    folder      = grafana_folder.example.id
    config_json = file("example.json")
}
```

### Expected Behavior
What should have happened?

We imported a pre-existing folder and dashboard into the Terraform state. With a previous Grafana version (v7.x) this always worked fine.

### Actual Behavior
What actually happened?

Now we're using Grafana 8. The import of the folder worked as before, but the import of the dashboard caused the following error:

```
│ Error: Cannot import non-existent remote object
│ 
│ While attempting to import an existing object to "grafana_dashboard.example", the provider detected that no object exists with the given id. Only pre-existing objects can be imported; check that the id is correct and that it is associated with
│ the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
```

For the import of the dashboard a slug is required. We took that from the URL and also double-checked that it is correct in Grafanas API /api/dashboards/uid/example-uid.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Manually create a dashboard in Grafana 8
2. `terraform import grafana_dashboard.example example-slug`

**Comment by JoschaLaubach** [comment]
Workaround: deleting the existing dashboard and having Terraform recreate it works. It's just the import that is broken here.

**Comment by mgfink** [comment]
Hello - I too am seeing the same problem trying to perform an import.  I'm running Terraform version 1.1.0 and AWS Managed  Grafana v8.2.5.

**Comment by mgfink** [comment]
I figured out my issue and thought I'd post here.  Instead of using the dashboard slug as I understood it (eg. given the following URL https://g-abcde1234b.grafana-workspace.us-east-1.amazonaws.com/d/EEQQHHHHH/unit, using "unit" as the slug) I used the key prior.

In the end the following command worked for me:
    
    terraform import grafana_dashboard.unit EEQQHHHHH 

tl;dr you can ignore my prior comment ;-)

**Comment by julienduchesne** [comment]
The import statement was fixed here: https://github.com/grafana/terraform-provider-grafana/commit/52eec2e03a9e46629573d1a975f41c7904d3dd36
